plugin/types.go

```diff
@@ -195,12 +195,10 @@ type Readable interface {
 	Read(context.Context) ([]byte, error)
 }
 
-// Writable is an entry that can write data directly to the entry. It mirrors
-// the WriterAt interface with an added context for handling the lifecycle of
-// remote write operations.
+// Writable is an entry that overwrites the prior contents
```

Comment:
Do you plan on updating the `Writable` docs separately? If not, then this should mention the last bullet point in https://github.com/puppetlabs/wash/issues/620#issuecomment-564192168 since the current comment is not true for GCP topics.

Reply:
I'll make some updates. I will need to write updates for the docs related to external plugins.